examples/test-convex-hull.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
```

Comment:
I added this example to compare two implementations and reverted all my changes of the example 50. 
Mostly it is copy/paste from the example 50. The difference is that ```obj1``` has 5 points and there is ```println``` of faces and points.